shared/constants/index.ts

```diff
@@ -6,3 +6,6 @@ export const MALTE_HELP_FORM_OFFER_ALIAS = 'help'
 export const APPOINTMENT_BOOKING_OFFER_ALIAS = 'terminbuchung'
 
 export const INTERNAL_OFFERS = [SPRUNGBRETT_OFFER_ALIAS, MALTE_HELP_FORM_OFFER_ALIAS]
+
+export const DATE_FORMAT = 'dd/MM/yyyy'
+export const INPUT_DATE_FORMAT = 'yyyy-MM-dd'
```

Comment:
Things from `shared` should not be imported directly. Instead, please export and import from `index.ts`.